client/chatbotWidget/src/components/HelpDeskWidget.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { sendMessageToChatbot } from '../chatbotApi.ts';
+
+// import app.css for styling/centering
+import '../App.css'
+
+const HelpDeskWidget: React.FC = () => {
+  // State to hold the current message input by the user
+  const [message, setMessage] = useState('');
+  // // State to hold the history of chat messages
+  const [chatHistory, setChatHistory] = useState<{ sender: string; text: string }[]>([]);
+
+  // Function to handle message from input box
+  const handlesSentMessage = async () => {
+    if (message.trim()){
+      try{
+        // Appends user message
+        setChatHistory((prevHistory: any) => [...prevHistory, { sender: 'You', text: message }]);
+
+        const botResponse = await sendMessageToChatbot(message)
+
+        // Appends bot response
+        setChatHistory((prevHistory: any) => [...prevHistory, { sender: 'Bot', text: botResponse }]);
+
+        setMessage('');
+      }
+      catch(error){
+        console.error("Error sending message:" , error)
+      }
+      setMessage('');
```

Comment:
Remove setMessage('') calls  in the try block and the one above  and add a "finally" block